pkg/queue/prometheus_stats_reporter.go

```diff
@@ -69,13 +65,14 @@ func newGV(n, h string) *prometheus.GaugeVec {
 
 // PrometheusStatsReporter structure represents a prometheus stats reporter.
 type PrometheusStatsReporter struct {
-	initialized bool
-	labels      prometheus.Labels
-	handler     http.Handler
+	initialized             bool
+	labels                  prometheus.Labels
+	handler                 http.Handler
+	reporterReportingPeriod time.Duration // ReporterReportingPeriod is the interval of time between reporting stats by queue proxy.
```

Comment:
```suggestion
	reportingPeriod time.Duration
```

Does the comment really add any explanation? Also please remove the stutter from the fieldname.

Reply:
Fixed